app/controllers/concerns/subdomainable.rb

```diff
@@ -1,6 +1,12 @@
 module Subdomainable
   extend ActiveSupport::Concern
 
+  included do
+    rescue_from ActiveRecord::RecordNotFound do |exception|
```

Comment:
Unused block argument - `exception`. You can omit the argument if you don't care about it.